### ISSUE TITLE
English

### ISSUE THREAD

**Issue by davidmoshal** [HcySunYang/vue-design#221]
Is there an English language version of this?

**Comment by davidmoshal** [comment]
offering $100 bounty to translate this to English.

**Comment by HcySunYang** [comment]
@davidmoshal 
Thank you for your interest in this article, but are you reading [For the reader](https://github.com/HcySunYang/vue-design/issues/167)? And unfortunately, for some reason, the pictures of this article are all lost, and the content of the article has been inconsistent with the current vue source.
In addition, I am very fortunate, now I am a vue3.0 collaborator, which means that I can see the source code. After vue3.0 is released, I will also write a set of articles, maybe written directly in English.

**Comment by davidmoshal** [comment]
yep, I read that, but I didn't really understand your point, maybe it was the translation.

I think I understand: 

a) you don't want to revisit your own writing on version 2.0 now that version 3.0 is coming out, and also, 

b) because you are a contributor, and the source is not public, perhaps you are not allowed to do so, correct?

**Comment by davidmoshal** [comment]
understood.
